JSDOMParser.js

```diff
@@ -278,7 +278,7 @@
 
   var whitespace = [" ", "\t", "\n", "\r"];
 
-  // See http://www.w3schools.com/dom/dom_nodetype.asp
+  // See https://www.w3schools.com/jsref/prop_node_nodetype.asp
```

Comment:
Can you replace this with https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType instead, and remove the other link entirely? Thank you!

Reply:
Upon your request, one of the links has been updated and the second one has been removed.